rash_book/src/vars.md

```diff
@@ -10,7 +10,7 @@ There are two kind of variables:
 
 ## Core
 
-By default, every execution of rash exposes two variables to the Context: `{{ rash }}` and `{{ env }}`.
+By default, every execution of Rash exposes two variables to the Context: `{{ rash }}` and `{{ env }}`.
 
```

Comment:
Same here, I prefer `rash` but it could be under discussion.

Reply:
Same, I think we could display it as code.

Reply:
yep